variables.tf

```
@@ -66,6 +66,18 @@ variable "resource" {
   description = "Hive resources"
 }
 
+variable "resource_proxy" {
+  type = object({
+    cpu     = number,
+    memory  = number
+  })
+  default = {
+    cpu     = 300,
+    memory  = 512
+  }
```

After a little more thought. The actual defaults in nomad are 128 memory, and 200 cpu I believe, maybe we should set that as default, and then you can change it for running where you need it! Sorry, I didn't think of this at first. 😅 
```suggestion
  default = {
    cpu     = 200,
    memory  = 128
  }
```

---

Done 👍